File: app/src/main/java/com/artmurka/artmurkaapp/model/pojo/itemlist/checkout/OrderContent.java

package com.artmurka.artmurkaapp.model.pojo.itemlist.checkout;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.HashMap;

public class OrderContent {

    @SerializedName("order_goods_fields")
    @Expose
    private OrderGoodsFields orderGoodsFields;
    @SerializedName("order_goods")
    @Expose
    private HashMap<String, OrderDesc> orderGoods;

    public OrderGoodsFields getOrderGoodsFields() {
        return orderGoodsFields;
    }

    public void setOrderGoodsFields(OrderGoodsFields orderGoodsFields) {
        this.orderGoodsFields = orderGoodsFields;
    }

    public HashMap<String, OrderDesc> getOrderGoods() {
        return orderGoods;
    }

    public void setOrderGoods(HashMap<String, OrderDesc> orderGoods) {
        this.orderGoods = orderGoods;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/checkout/OrderContent.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.checkout

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.util.HashMap

class OrderContent {

    @SerializedName("order_goods_fields")
    @Expose
    var orderGoodsFields: OrderGoodsFields? = null
    @SerializedName("order_goods")
    @Expose
    var orderGoods: HashMap<String, OrderDesc>? = null

}
